dist/rpm/os-autoinst.spec

```
@@ -85,10 +85,13 @@ Source0:        %{name}-%{version}.tar.xz
 %else
 %define python_style_requires %{nil}
 %endif
-%ifnarch ppc ppc64 ppc64le s390x
-%bcond_without ocr
-%else
+%ifarch ppc ppc64 ppc64le s390x
+%define ocr_not_supported_by_arch 1
+%endif
+%if 0%{ocr_not_supported_by_arch} || !0%{?is_opensuse}
```

From https://rpm-software-management.github.io/rpm/manual/macros.html

```suggestion
%ifarch ppc ppc64 ppc64le s390x
%define ocr_not_supported_by_arch 1
%endif
%if 0%{?ocr_not_supported_by_arch} || !0%{?is_opensuse}
```

Or, looking at https://github.com/rpm-software-management/rpm/blob/master/docs/manual/conditionalbuilds.md#creating-build-conditionals you might probably use something like 

```suggestion
%ifarch ppc ppc64 ppc64le s390x
%bcond ocr_not_supported_by_arch 1
%endif
%if %{with ocr_not_supported_by_arch} || !0%{?is_opensuse}
```

_Note: this is syntax suggestion only, I did not try to understand the full logic workflow to optimize the conditionals._